src/parser/pegen.ts

```diff
@@ -11,7 +11,7 @@ import {
     Constant,
     exprKind,
 } from "../ast/astnodes.ts";
-import { TokenInfo } from "../tokenize/tokenize.ts";
+import type { TokenInfo } from "../tokenize/tokenize.ts";
```

Comment:
what's this magic?

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html
It got suggested to me when I tried to import a type somewhere else.
It basically says - "when it comes to bundling you can just ignore me"

Reply:
Sweet, this *could* help with skulpt bundling